spec/models/casa_org_spec.rb

```diff
@@ -128,52 +181,56 @@
       end
     end
 
-    describe "mileage rate for a given date" do
-      let(:casa_org) { build(:casa_org) }
+    it "generates default contact topics" do
+      expect(org.contact_topics).to eq(contact_topics)
+    end
+  end
```

Comment:
This looks like a lot of changes but I simply moved an `end` to fix nesting. Prev it went like 
```
desc 'defaults' do
   desc 'mileage' do
   end
end
```
now its
```
desc 'defaults' do end
desc 'mileage' do end
```